guide/05-working-with-the-spatially-enabled-dataframe/part3_data_io_writing_data.ipynb

```diff
@@ -0,0 +1,1816 @@
+{
```

Comment:
<p>In this guide, we explored how <a href="https://developers.arcgis.com/python/api-reference/arcgis.features.toc.html#geoaccessor" target="_blank"><strong>Spatially enabled DataFrame</strong></a> (SeDF) can be used to export spatial data to various formats. We started by exporting the data to web feature layers and to in-memory JSON based formats, such as FeatureSet and FeatureCollection. Next, we explored writing the data to various local data sources, such as a file geodatabase, a mobile geodatabase, an sqlite database, and a shapefile. We also discussed exporting the data to non-spatial formats, such as a csv file or a table. We introduced how the data in local file formats, such as a feature class or a table in a File Geodatabase, can be overwritten using a SeDF. Towards the end, we discussed how the data from SeDF can be exported to in-memory workspaces. </p><p>In the next part of the guide series, you will learn about the various properties of a SeDF and how they can be used to pre-process a SeDF.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='92'/>